crates/btc-notify/src/subscription.rs

```diff
@@ -13,6 +13,12 @@ use tokio::sync::mpsc;
 pub struct Subscription<T> {
     receiver: mpsc::UnboundedReceiver<T>,
 }
+impl<T> Subscription<T> {
+    /// Returns the number of messages in the backlog for this subscription.
+    pub fn backlog(&self) -> usize {
```

Comment:
Can this be `const`?
```suggestion
    pub const fn backlog(&self) -> usize {
```